### ISSUE TITLE
[Requesting comment] Smallest possible XML file

### ISSUE THREAD

**Issue by Zegnat** [mathiasbynens/small#32]
By strict reading of the [XML 1.0 standard](http://www.w3.org/TR/REC-xml/) am I right to assume the smallest possible XML file is the following?

``` xml
<?xml version='1.0'?><_/>
```

This does validate as ‘well-formed XML’ on [the W3C Markup Validator](http://validator.w3.org). The only odd part is that the root element is seen as `_/` rather than `_`. Looks like a bug, because it does treat the slash as a closing slash.

Not sure if I should pull, requesting comment.

---

A XML document is:
```
document      ::=   prolog element Misc*
```
Where:
```
prolog        ::=   XMLDecl? Misc* (doctypedecl Misc*)?
XMLDecl       ::=   '<?xml' VersionInfo EncodingDecl? SDDecl? S? '?>'
VersionInfo   ::=   S 'version' Eq ("'" VersionNum "'" | '"' VersionNum '"')
Eq            ::=   S? '=' S?
VersionNum    ::=   '1.' [0-9]+
```
And:
```
element       ::=   EmptyElemTag
                    | STag content ETag
```
With:
```
EmptyElemTag  ::=   '<' Name (S Attribute)* S? '/>'
```
While:
```
Name          ::=   NameStartChar (NameChar)*
```
In short we get this when all optional (marked by `?` or `*`) parts are left out:
```
document      ::= '<?xml' S 'version' '=' ("'" '1.' [0-9]+ "'" | '"' '1.' [0-9]+ '"') '?>' '<' NameStartChar '/>'
```

**Comment by waldyrious** [comment]
@mathiasbynens shouldn't this be closed after the resolution of #34?

**Comment by mathiasbynens** [comment]
@waldyrious You’re right. Thanks!
